config/environments/development.rb

```diff
@@ -14,6 +14,9 @@
   # Show full error reports.
   config.consider_all_requests_local = true
 
+  # Enable server timing
+  config.server_timing = true
```

Comment:
[Auto generated change]
On development environment, we can check performance metrics from the server on the browser's developer tools.
I added the documentation although it is Rails Edge version.
- https://edgeguides.rubyonrails.org/configuring.html#config-server-timing